src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+
+  border: 1vmin #000;
+  border-style: dotted;
+  border-radius: 50%;
+
+  height: 80vmin;
+  width: 80vmin;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minutes {
+    position: absolute;
+    bottom: 50%;
+
+    height: 20vmin;
+    width: 3vmin;
+
+    background-color: #0700ff;
+    transform-origin: bottom center;
+
+    animation: rotation 3600s steps(60) infinite;
+
+    @include rotation-clock;
+  }
+
+  &__seconds {
+    position: absolute;
+    bottom: 50%;
+
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    transform-origin: bottom center;
+
+    animation: rotation 60s infinite linear;
+
+    @include rotation-clock;
```

Comment:
Similar to the previous comment, the `@include rotation-clock;` directive is used here without a definition or import of the `rotation-clock` mixin. Ensure it is defined or imported correctly.